core/templates/domain/admin/admin-backend-api.service.ts

```diff
@@ -538,6 +538,13 @@ export class AdminBackendApiService {
         });
     });
   }
+
+  async generateDummyBlogAsync(blog: string): Promise<void> {
```

Comment:
blog --> blog post, please fix here and elsewhere

Reply:
Done.